HACKING.md

```diff
@@ -13,16 +13,16 @@ Main dependencies:
 On windows above dependencies are resolved automatically during cmake phase but you must also have Windows SDK v10.0.20348.0 or higher installed.
 
 Additional dependencies:
-* CMake 3.20.4+ (3.20.21032501-MSVC_2 from VS2019 won't work)
+* CMake 3.20.4+ (3.24.202208181-MSVC_2 from VS2022 won't work)
```

Comment:
Eh, I'm failing to parse this.

We require 3.20.4+, but 3.24.xxxx won't work?

Reply:
Will test 3.24.202208181-MSVC_2 today and report. Depending on outcome this either should be left as it was before or reworded to something like "cmake versions bundled with VS2019/VS2022 won't work, you need to install standalone cmake from official site).

Reply:
Tested cmake 3.24.202208181-MSVC_2 and it is working just fine. We need to revert this to how it was before.

Reply:
reverted